src/consensus/network_event.rs

```diff
@@ -29,16 +29,16 @@ pub struct AckMessagePayload {
     pub dst_name: XorName,
     /// The prefix of our section when we acknowledge their SectionInfo of version ack_version.
     pub src_prefix: Prefix<XorName>,
-    /// The version acknowledged.
-    pub ack_version: u64,
+    /// The key acknowledged.
+    pub ack_key: bls::PublicKey,
 }
 
 #[derive(Debug, Serialize, Deserialize, PartialEq, Eq, PartialOrd, Ord, Hash, Clone)]
 pub struct SendAckMessagePayload {
```

Comment:
I think this whole Ack approach can be deleted here actually. Perhaps when the `pull` mechanism in place would be more correct. Happy to waiton that PR

Reply:
Yes, that will be part of the pull model.